tests/system/datastore/test_redis.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import mlrun.datastore
+from tests.system.base import TestMLRunSystem
+
+
+class TestRedisDataStore(TestMLRunSystem):
```

Comment:
Usually in system tests we would use `@tests.system.base.TestMLRunSystem.skip_test_if_env_not_configured` to decorate the class or specific tests, to verify that a system is configured as pre-requisite to run the tests.
In this case I assume you need redis to be configured in the system you connect to, is there any specific way to test that, or whether it's always fine to run the tests?

Reply:
if redis is not configured then fakeRedis (python mockup of redis) is used - so no need to check..